src/components/CalendarLayout/_components/CalendarList/CalendarList-styles.less

```diff
@@ -27,8 +27,17 @@
 	}
 }
 
+:where(.css-dev-only-do-not-override-u24u6l).ant-segmented .ant-segmented-group {
```

Comment:
N'utilise pas les classes `.css-dev-only-do-not-override`. Le nom de la classe est assez explicite, il est indiqué de ne pas les utiliser pour appliquer notre propre style.